packages/kotti-ui/source/kotti-style/forms/_toggles.scss

```diff
@@ -0,0 +1,65 @@
+.form-radio {
+	position: relative;
+	display: inline-block;
+	min-height: 1.2rem;
+	padding: (($control-size-sm - $line-height) / 2) $control-padding-x
+		(($control-size-sm - $line-height) / 2)
+		($control-icon-size + $control-padding-x);
+	margin: ($control-size - $control-size-sm) / 2 0;
+	line-height: $line-height;
+
+	input {
+		position: absolute;
+		width: 1px;
+		height: 1px;
+		margin: -1px;
+		overflow: hidden;
+		clip: rect(0, 0, 0, 0);
+
+		&:focus + .form-icon {
+			border-color: var(--interactive-01);
+		}
+
+		&:checked + .form-icon {
+			background: var(--interactive-01);
+			border-color: var(--interactive-01);
+		}
+		&:invalid + .form-icon {
+			border: 1px solid var(--support-error);
+		}
+	}
+
+	.form-icon {
+		position: absolute;
+		display: inline-block;
+		cursor: pointer;
+		border: $border-width solid var(--ui-02);
+	}
+
+	// Input checkbox, radio and switch sizes
+	&.input-sm {
+		margin: 0;
+		font-size: $font-size-sm;
+	}
+
+	&.input-lg {
+		margin: ($control-size-lg - $control-size-sm) / 2 0;
+		font-size: $font-size-lg;
+	}
+}
+
+.form-radio {
+	.form-icon {
+		top: ($control-size-sm - $control-icon-size) / 2;
```

Comment:
@carsoli can you maybe check if some of those SCSS variables are dead now that the classes are gone?